### PR TITLE
Only use peer data when Plug is a dependency

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
In practice this should not happen, but removes the noisy compilation warning when using mv_opentelemetry as a dependency and it gets compiled before Plug.

Signed-off-by: Maciej Szlosarczyk <maciej@mindvalley.com>

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **maciej-szlosarczyk** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by antonsatin — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 65f3071eff2e0cb7401407f919ab58041d54c610</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/26#pullrequestreview-885878472" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
New changes are detected. LGTM label has been removed.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **antonsatin** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **antonsatin** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/mindvalley/mv-opentelemetry/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [f9022a2](https://github.com/mindvalley/mv-opentelemetry/commits/f9022a2dfead8ea747c235bd0570f30311867fe9) Specify correct events to receive in tests

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by antonsatin — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 4e213f97e62102712f954aca736aa68a00b84e4c</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/26#pullrequestreview-909778633" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
